src/AutoRest.CSharp/DataPlane/Output/DataPlaneResponseHeaderGroupType.cs

```diff
@@ -22,6 +22,7 @@ internal class DataPlaneResponseHeaderGroupType: TypeProvider
 
         public DataPlaneResponseHeaderGroupType(OperationGroup operationGroup, Operation operation, HttpResponseHeader[] httpResponseHeaders, BuildContext<DataPlaneOutputLibrary> context) : base(context)
         {
+            DefaultAccessibility = "internal";
```

Comment:
nit: move next to the default name assignment

Reply:
obsolete given comment below